pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -240,6 +215,25 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, const
 	return output.LaunchTemplate, nil
 }
 
+func (p *LaunchTemplateProvider) getBlockDeviceMappings(constraints *v1alpha1.Constraints) []*ec2.LaunchTemplateBlockDeviceMappingRequest {
```

Comment:
consider following the non-getter pattern established elsewhere (e.g. launchTemplateName)
```suggestion
func (p *LaunchTemplateProvider) blockDeviceMappings(constraints *v1alpha1.Constraints) []*ec2.LaunchTemplateBlockDeviceMappingRequest {
```